studio/app/common/routers/workflow.py

```diff
@@ -87,3 +88,28 @@ async def import_workflow_config(file: UploadFile = File(...)):
         return WorkflowConfig(**contents)
     except Exception as e:
         raise HTTPException(status_code=400, detail=f"Parsing yaml failed: {str(e)}")
+
+
+@router.get(
+    "/sample_data",
```

Comment:
endpoint would be `/sample_data/{workspace_id}`.
retrieve `workspace_id` as path parameter.